### DISCUSSION TITLE
How to disable useDrag or useGesture

### DISCUSSION THREAD

**Discussion by pailhead** [pmndrs/use-gesture#635] [Q&A]
Hi, I've encountered this library for the first time in a new project.

I would like to disable the drag gesture under specific circumstances. For example i am able to do what i need with

```
useGesture({
  onDrag:()=>{
    if(foo) return
  }
})
```

However, when i try the `enabled` flag absolutely nothing happens:
```
useGesture({
  onDrag:()=>{}
},
{
   enabled: !foo
})
```

Is this a bug or a user error? The comment for `enabled` makes it seem like it should do this. Thank you

**Comment by dbismut** [comment]
Can you post the rest of the code?
